docs/nirfsg/enums.rst

```diff
@@ -1441,92 +1468,76 @@ ReferenceClockRate
 
 
 
-RelativeTo
-----------
-
-.. py:class:: RelativeTo
-
-    .. py:attribute:: RelativeTo.CURRENT_POSITION
+    .. py:attribute:: ReferenceClockExportedRate._100mhz
 
 
 
-        The reference position is relative to the current position.
+        Uses a 100MHz Reference Clock rate.
 
         
 
 
 
-    .. py:attribute:: RelativeTo.START_OF_WAVEFORM
+    .. py:attribute:: ReferenceClockExportedRate._1ghz
 
 
 
-        The reference position is relative to the start of the waveform.
+        Uses a 1GHz Reference Clock rate.
 
         
 
 
 
-ResetOptions
```

Comment:
FYI, the reason this enum is excluded is that it's not used anywhere, most likely because [reset_with_options](https://www.ni.com/docs/en-US/bundle/ni-rfsg-c-api-ref/page/cvirfsg/cvinirfsg_resetwithoptions.html) isn't being included in the API. Not sure if that function is important to you.